WordPress/AbstractClassRestrictionsSniff.php

```diff
@@ -116,7 +116,7 @@ public function process_token( $stackPtr ) {
 	 *
 	 * @return bool
 	 */
-	public function is_targetted_token( $stackPtr ) {
+	public function is_targeted_token( $stackPtr ) {
```

Comment:
There is a public method name change here.

Reply:
From a quick search on the GH there are some external standards which seem to be using this abstract class, so changing this public method would be a major BC break IIRC.

Reply:
Okay, reverting ...

Reply:
Done 🍏